apm-sniffer/apm-sdk-plugin/spring-plugins/mvc-annotation-commons/src/main/java/org/apache/skywalking/apm/plugin/spring/mvc/commons/ReactiveRequestHolder.java

```diff
@@ -37,7 +38,11 @@ public String getHeader(final String headerName) {
 
     @Override
     public Enumeration<String> getHeaders(final String headerName) {
-        return Collections.enumeration(this.serverHttpRequest.getHeaders().get(headerName));
+        List<String> values = this.serverHttpRequest.getHeaders().get(headerName);
+        if (values == null) {
+            return null;
```

Comment:
Is null expected  when use this method? You used to say returning an empty list.